packages/docker/src/executors/release-publish/schema.d.ts

```diff
@@ -0,0 +1,10 @@
+export interface DockerReleasePublishSchema {
+  dryRun?: boolean;
+  quiet?: boolean;
+}
+
+export interface NormalizedDockerReleasePublishSchema {
```

Comment:
Can we inline this in the file? I was initially confused by its presence here and why imageReference was not on the schema.json, I would say this type is really just implementation details related and not the schema directly